inst/templates/data-table.R

```diff
@@ -0,0 +1,14 @@
+#' data.table is generally careful to minimize the scope for namespace
```

Comment:
I don't _think_ that you want this text as a roxygen2 comment, as that will cause a warning when the downstream user goes to document (since there's no topic name), and it's mostly aimed at the user of usethis, not the user of the package it's applied to.

Reply:
Is there anything to put in the template file then? Feels strange to leave it blank. I saw a lot of documentation in the `tidy-eval.R` template is why I felt the need to add something. But I guess also `use_tidy_eval()` is _exporting_ things too, which `use_data_table()` is not.

Reply:
ooh gotcha. yes that was the intent.